### PR TITLE
Develop ck hibachi updates

### PR DESCRIPTION
**Pull request by christopher-kundrat**
This change is [Reviewable](https://reviewable.io/reviews/ten24/slatwall/5386)

### CODE REVIEW

File: org/Hibachi/HibachiTags/HibachiPropertyTable.cfm [hunk 1]
```diff
@@ -1,4 +1,3 @@
-<cfimport prefix="swa" taglib="../../../tags" />
```
Comment (rmarchandTen24) [inline org/Hibachi/HibachiTags/HibachiPropertyTable.cfm]: @sumitverma @gregmoser 
removing swa from hibachi tags makes sense to me. Is there any reason not to remove them?

File: org/Hibachi/HibachiJsonService.cfc [hunk 2]
```diff
@@ -55,7 +55,7 @@ component extends="HibachiService" persistent="false" accessors="true" output="f
 		config[ 'modelConfig' ] = getModel();
 		json['data'] = config;
 		json = serializeJson(json);
-		var filePath = expandPath('/Slatwall') & '/custom/config/config.json';
+		var filePath = expandPath('/#getDao("HibachiDao").getApplicationKey()#') & '/custom/config/config.json';
```
Comment (rmarchandTen24) [inline org/Hibachi/HibachiJsonService.cfc]: did you try using this in hibachicore @christopher-kundrat

File: org/Hibachi/HibachiTags/HibachiPropertyTable.cfm (hunk 1 above)
Comment (sumitverma) [inline org/Hibachi/HibachiTags/HibachiPropertyTable.cfm]: makes sense.

File: org/Hibachi/HibachiJsonService.cfc (hunk 2 above)
Comment (christopher-kundrat) [inline org/Hibachi/HibachiJsonService.cfc]: @rmarchandTen24 - yeah I tested it there and everything looked good

**Pull request merged**